cloudstack/data_source_cloudstack_vpc.go

```diff
@@ -37,6 +37,11 @@ func dataSourceCloudstackVPC() *schema.Resource {
 		Schema: map[string]*schema.Schema{
 			"filter": dataSourceFiltersSchema(),
 
+			"projectid": {
```

Comment:
We already have a field for `project`. It will be better to use that instead of adding a new field.

Reply:
`Optional: true,` helps

however, then we need to get the project (by name) at first, then list resources in the project